src/common/types/timestamp_t.cpp

```diff
@@ -256,12 +256,20 @@ void Timestamp::convert(timestamp_t timestamp, date_t& out_date, dtime_t& out_ti
     out_time = getTime(timestamp);
 }
 
-timestamp_t Timestamp::fromEpochMs(int64_t epochMs) {
-    return timestamp_t(epochMs * Interval::MICROS_PER_MSEC);
+timestamp_t Timestamp::fromEpochMicroSeconds(int64_t micros) {
+    return timestamp_t(micros);
 }
 
-timestamp_t Timestamp::fromEpochSec(int64_t epochSec) {
-    return timestamp_t(epochSec * Interval::MICROS_PER_SEC);
+timestamp_t Timestamp::fromEpochMilliSeconds(int64_t ms) {
+    return fromEpochMicroSeconds(ms * Interval::MICROS_PER_MSEC);
+}
+
+timestamp_t Timestamp::fromEpochSeconds(int64_t sec) {
+    return fromEpochMicroSeconds(sec * Interval::MICROS_PER_SEC);
+}
+
+timestamp_t Timestamp::fromEpochNanoSeconds(int64_t ns) {
```

Comment:
Is there any test covering this? `fromEpochNanoSeconds` If not, let's add one for it!

Reply:
Pyarrow doesn't automatically convert the nano seconds timestamp to micro seconds...
I can't find a tool that generates a nano second timestamp column